### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by waigani**
Adding a comment with no space after "//".

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
 package main
 
+//no space after forward slash
 func main(){
 }
```
Comment (waigani) [inline main.go]: Test comment.

File: main.go [hunk 2]
```diff
@@ -1,4 +1,5 @@
 package main
 
+//no space after forward slash
 func main(){
```
Comment (lingo-reviews) [inline main.go]: Another test comment.

File: main.go [hunk 3]
```diff
@@ -1,4 +1,7 @@
 package main
```
Comment (lingo-reviews) [inline main.go]: New comment

Comment (mrpedantic) [inline main.go]: new comment from lingo

File: main.go [hunk 4]
```diff
@@ -1,4 +1,7 @@
...
```
Comment (lingo-reviews) [inline main.go]: new comment really from lingo

Comment (lingo-reviews) [inline main.go]: new comment really from lingo

**Pull request closed without merging**